src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -22,15 +20,14 @@
 #include <validation.h>
 #include <validationinterface.h>
 
-#include <tinyformat.h>
```

Comment:
What's the rationale behind these include updates? According to `iwyu` this include is still necessary. More generally, I think it's most productive to either fix all the includes, or the ones that are affected by the PR, but not a (seemingly random) subset?